chapters/05_chapter_4.tex

```diff
@@ -0,0 +1 @@
+\chapter{Replace with Chapter 4 Name}
```

Comment:
Why are you adding this? This is for other students to add, please remove all such files. also from the main

Reply:
Done